src/uct/api/v2/uct_v2.h

```diff
@@ -212,7 +217,7 @@ uct_iface_estimate_perf(uct_iface_h tl_iface, uct_perf_attr_t *perf_attr);
  * @brief Undo the operation of @ref uct_md_mem_reg() and invalidate memory
  *        region.
  *
- * @param [in]  md          Memory domain which was used to register the memory.
```

Comment:
Please add full description in addition to brief description.

Reply:
added